### PR TITLE
Css two column layout -Tim Chiang

### PR DESCRIPTION
**Pull request by tychiang922**
image

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Tim! A couple notes:
1. Your line spacing on the subheader in the first row and the text in the second row is incorrect as compared to the example.
2. There is too much space between the columns horizontally in the second row.

Please revise and submit a new screenshot with your revision.

**Comment by tychiang922** [comment]
image

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Your line spacing is still incorrect

**Comment by tychiang922** [comment]
> Your line spacing is still incorrect

Line spacing as in line height for the subheader and text-content?

**Comment by tychiang922** [comment]
image

**Comment by tychiang922** [comment]
image

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                  across Africa - making them radically more efficient
+                  operationally and financially, a complete turnaround.
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="col-2 prl-5perc ptb-60-20">
+          <div class="row">
+            <div class="col-2-1fourth">
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:47]: `col-2-1fourth`
I don't see a rule for this class.

File: css-two-column-layout/index.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+              <img src="images/practice.svg" alt="practice">
+            </div>
+            <div class="col-2-3fourth pt-30">
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:50]: `col-2-3fourth`
I don't see a rule for this class.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by tychiang922** [comment]
image

**Review by codingcodymiller — APPROVED** [review]
Nice work!!

**Pull request merged**